### PR TITLE
Created climber code for Grogu

### PR DESCRIPTION
**Pull request by ngupta-22**

### CODE REVIEW

**Review by ngupta-22 — COMMENTED** [review]
Looks good! I'm a little confused about how the braking works though. Are the CL_STOPPED/NOT_STOPPED buttons on smart dashboard just indicators or do they control something?

**Review by ashanawaz — APPROVED** [review]
Looks good

**Comment by jmullins16** [comment]
I think Christine is in the process of debugging the brake, so there are some things commented out or slightly unfinished. Including the dashboard commands. I asked her to commit and push to the repo so others could review the code, but the PR doesn't need to be merged until she gets those a little closer.

File: 2021/OffSeason2021/src/main/cpp/subsystems/Climber.cpp [hunk 1]
```diff
@@ -125,29 +125,31 @@ void Climber::RaiseClimberWithJoysticks(frc::XboxController *joystick)
...
+    {
+        SetBrakeSolenoid(CL_BRAKE_UNLOCKED);
+        // If joystick is above a value, climber will move up
+        if (yCLValue > 0.1)
+            motorOutput = m_upSpeed;
+
+        // If joystick is below a value, climber will move down
+        else if (yCLValue < -0.1)
+            motorOutput = m_downSpeed;
+    }
```
Comment (ngupta-22) [inline 2021/OffSeason2021/src/main/cpp/subsystems/Climber.cpp:143]: Don't forget to square the motor output and keep the original sign after squaring

Comment (cgjyang) [reply 2021/OffSeason2021/src/main/cpp/subsystems/Climber.cpp:143]: Will do that as soon as I can!

Comment (jmullins16) [reply 2021/OffSeason2021/src/main/cpp/subsystems/Climber.cpp:143]: Don't wait for the squaring to get this PR approved and in. Treat that as a separate issue. This is exactly what I mean about getting a base level of functionality, making a commit, creating a PR, and getting it merged. We don't want to be waiting for the final version when we have one that gives us valuable base functionality.

**Pull request merged**